src/Microsoft.AspNetCore.HttpOverrides/ForwardedHeadersOptions.cs

```diff
@@ -67,6 +67,20 @@ public class ForwardedHeadersOptions
         /// </summary>
         public IList<IPNetwork> KnownNetworks { get; } = new List<IPNetwork>() { new IPNetwork(IPAddress.Loopback, 8) };
 
+        /// <summary>
+        /// The allowed values from x-forwarded-host. If the list is empty then all hosts are allowed.
+        /// Failing to restrict this these values may allow an attacker to spoof links generated by your service. 
+        /// </summary>
+        /// <remarks>
+        /// - Port numbers must be excluded
```

Comment:
I don't think this will be formatted correctly in doc comments. There's some kind of `<list>` element you need to use. (Kind of like HTML, but different.)